docs/src/apireference.md

```diff
@@ -96,47 +96,18 @@ The value of the attribute is of type `ResultStatusCode`.
 ResultStatusCode
 ```
 
+## Variables and Constraints
+
 ### Basis Status
 
-[what? BasisStatus]
+The `BasisStatus` attribute of a variable or constraint indicates its presence in a basis.
```

Comment:
fixed but i still think more thought needs to go into these. the basis status description in attributes.jl is not super clear. eg what if it's not an optimal primal solution, but rather a ray